bccsp/schemes/idemixopts.go

```diff
@@ -296,15 +299,26 @@ func (o *IdemixSignerOpts) HashFunc() crypto.Hash {
 
 type EidNymAuditOpts struct {
 	AuditVerificationType AuditVerificationType
-	EidIndex              int
-	EnrollmentID          string
-	RNymEid               *math.Zr
+	EidIndex              int      // Index of enrollment ID attribute in signature
+	EnrollmentID          string   // Enrollment ID of identity
+	RNymEid               *math.Zr // Field element of randomness
 }
 
 func (o *EidNymAuditOpts) HashFunc() crypto.Hash {
 	return 0
 }
 
+type RhNymAuditOpts struct {
```

Comment:
Please add comments for the struct

Reply:
same as my comment about comments for functions, comments for structs need to start with the struct name